### PR TITLE
mapped input names and tensors directly rather than copying which was…

### PR DESCRIPTION
**Pull request by sfatimar**
… leading to mismatch

Directly mapping the input map to tensors so that name mapping can be done directly 
Disabling test_mvn as it is not being used by any major models 

Motivation
- This changes were required to fix test_batchnorm_example_cpu in which input was being rerranged alphabetically

### CODE REVIEW

File: onnxruntime/core/providers/openvino/backend_manager.cc [hunk 1]
```diff
@@ -35,11 +35,11 @@ BackendManager::BackendManager(const onnxruntime::Node* fused_node, const loggin
 
   // Save the indexes of graph inputs among fused_node's inputDefs
   // (which also contains initializers).
-  std::map<std::string, int> inputdef_index_map;
+  //std::map<std::string, int> inputdef_index_map;
```
Comment (smkarlap) [inline onnxruntime/core/providers/openvino/backend_manager.cc:38]: Remove all commented code lines.

**Comment by smkarlap** [comment]
LGTM

**Comment by suryasidd** [comment]
LGTM. i will merge this after the commented lines are removed

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc [hunk 2]
```diff
@@ -87,24 +87,27 @@ void BasicBackend::StartAsyncInference(Ort::CustomOpApi& ort,
     auto graph_input_buffer = graph_input_blob->buffer()
                                   .as<InferenceEngine::PrecisionTrait<InferenceEngine::Precision::FP32>::value_type*>();
     size_t input_data_size = graph_input_blob->byteSize();
-
-    auto tensor_shape = ort.GetTensorTypeAndShape(input_tensors[i]);
+    
+    std::string input_name = input_info_iter->first;
```
Comment (suryasidd) [inline onnxruntime/core/providers/openvino/backends/basic_backend.cc:91]: Move this to line to 78. The variable input_name can be re-used in multiple places

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc [hunk 3]
```diff
@@ -87,24 +87,27 @@ void BasicBackend::StartAsyncInference(Ort::CustomOpApi& ort,
...
+    const OrtValue* tensor = ort.KernelContext_GetInput(context, subgraph_context_.input_names.at(input_name));
```
Comment (suryasidd) [inline onnxruntime/core/providers/openvino/backends/basic_backend.cc:92]: Shouldn't these changes be added to vad-m backend as well

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc [hunk 4]
```diff
@@ -179,11 +182,10 @@ void BasicBackend::Infer(Ort::CustomOpApi& ort, OrtKernelContext* context) {
   std::lock_guard<std::mutex> lock(compute_lock_);
 
   size_t batch_size = 1;
-  // Get Input and Output tensors
-  auto input_tensors = GetInputTensors(ort, context, ie_cnn_network_, subgraph_context_.input_indexes);
```
Comment (suryasidd) [inline onnxruntime/core/providers/openvino/backends/basic_backend.cc:183]: If we are not going to use GetInput Tensors, we should remove it

**Comment by suryasidd** [comment]
> LGTM. i will merge this after the commented lines are removed

Sorry, missed a few comments

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc (hunk 4 above)
Comment (sfatimar) [reply onnxruntime/core/providers/openvino/backends/basic_backend.cc:183]: > Done

File: onnxruntime/core/providers/openvino/backend_manager.cc (hunk 1 above)
Comment (sfatimar) [reply onnxruntime/core/providers/openvino/backend_manager.cc]: Done

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc (hunk 2 above)
Comment (sfatimar) [reply onnxruntime/core/providers/openvino/backends/basic_backend.cc]: removed the variable all together

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc (hunk 3 above)
Comment (sfatimar) [reply onnxruntime/core/providers/openvino/backends/basic_backend.cc]: done

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc (hunk 4 above)
Comment (sfatimar) [reply onnxruntime/core/providers/openvino/backends/basic_backend.cc:183]: done

File: onnxruntime/core/providers/openvino/backends/basic_backend.cc [hunk 5]
```diff
@@ -87,24 +87,26 @@ void BasicBackend::StartAsyncInference(Ort::CustomOpApi& ort,
     auto graph_input_buffer = graph_input_blob->buffer()
                                   .as<InferenceEngine::PrecisionTrait<InferenceEngine::Precision::FP32>::value_type*>();
     size_t input_data_size = graph_input_blob->byteSize();
-
-    auto tensor_shape = ort.GetTensorTypeAndShape(input_tensors[i]);
+    
+    const OrtValue* tensor = ort.KernelContext_GetInput(context, subgraph_context_.input_names.at(input_info_iter->first));
```
Comment (suryasidd) [inline onnxruntime/core/providers/openvino/backends/basic_backend.cc:91]: If we are using input_info_iter->first 4 times, it's better to have a variable for that right? It makes the code look cleaner.

File: onnxruntime/core/providers/openvino/backends/vadm_backend.cc [hunk 6]
```diff
@@ -130,8 +130,9 @@ void VADMBackend::StartAsyncInference(Ort::CustomOpApi& ort, std::vector<const O
     auto graph_input_buffer =
         graph_input_blob->buffer().as<InferenceEngine::PrecisionTrait<InferenceEngine::Precision::FP32>::value_type*>();
 
+    const OrtValue* tensor = ort.KernelContext_GetInput(context, subgraph_context_.input_names.at(input_info_iter->first));
```
Comment (suryasidd) [inline onnxruntime/core/providers/openvino/backends/vadm_backend.cc:133]: Same comment here as well

File: onnxruntime/core/providers/openvino/backends/vadm_backend.cc [hunk 7]
```diff
@@ -130,8 +130,9 @@ void VADMBackend::StartAsyncInference(Ort::CustomOpApi& ort, std::vector<const O
...
     size_t input_data_size = graph_input_blob->byteSize();
-    const char* tensor_data = ort.GetTensorData<char>(input_tensors[i]);
+    const char* tensor_data = ort.GetTensorData<char>(tensor);
     const char* batch_memory_offset = tensor_data + input_data_size * batch_slice_idx;
```
Comment (suryasidd) [inline onnxruntime/core/providers/openvino/backends/vadm_backend.cc:136]: Shouldn't we add Int64 logic here as well ?

**Comment by suryasidd** [comment]
Lets include the remaining changes in another patch

**Pull request merged**